pkg/riff/commands/riff.go

```diff
@@ -40,6 +40,7 @@ func NewRiffCommand(ctx context.Context, c *cli.Config) *cobra.Command {
 	cmd.AddCommand(NewHandlerCommand(ctx, c))
 	cmd.AddCommand(NewStreamCommand(ctx, c))
 	cmd.AddCommand(NewProcessorCommand(ctx, c))
+	cmd.AddCommand(NewDoctorCommand(ctx, c))
```

Comment:
I moved the doctor command to be attached to the [root command](https://github.com/projectriff/cli/blob/e5a90713536175ecedaece277e0a98df49ca636e/pkg/riff/commands/root.go#L49). The root command is the riff command, however, the difference is that the riff command is designed to be embedded into other command hierarchies whereas the root command should only be used by the riff cli.